creds/ssm/manager.go

```diff
@@ -0,0 +1,123 @@
+package ssm
+
+import (
+	"errors"
+	"io/ioutil"
+	"os"
+	"text/template"
+	"text/template/parse"
+
+	"code.cloudfoundry.org/lager"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/concourse/atc/creds"
+)
+
+const DefaultPipeSecretTemplate = "/concourse/{{.Team}}/{{.Pipeline}}/{{.Secret}}"
+const DefaultTeamSecretTemplate = "/concourse/{{.Team}}/{{.Secret}}"
+
+type SsmManager struct {
+	AwsAccessKeyID     string `long:"aws-access-key" description:"AWS Access key ID"`
```

Comment:
These `aws-` prefixes can probably be removed along with the namespace change to `aws-ssm` mentioned (far) below.